### PR TITLE
Added output_file parameters, fixed some readme formatting

### PR DESCRIPTION
**Pull request by tmuntianu**

### CODE REVIEW

File: supereeg/brain.py [hunk 1]
```diff
@@ -418,7 +418,7 @@ def plot_data(self, filepath=None, time_min=None, time_max=None, title=None,
 
         if self.get_data().shape[0] == 1:
             nii = self.to_nii()
-            nii.plot_glass_brain()
+            nii.plot_glass_brain(pdfpath=filepath)
```
Comment (jeremymanning) [inline supereeg/brain.py]: I think this should be `output_file`

Comment (tmuntianu) [reply supereeg/brain.py]: `nii.plot_glass_brain()` calls `plot_glass_brain()` in on a SuperEEG nifti object, which takes `pdfpath`

Then, nifti's `plot_glass_brain()` calls `nilearn.plot_glass_brain()` which takes `output_file`

Comment (jeremymanning) [reply supereeg/brain.py]: got it 👌

**Comment by lucywowen** [comment]
Looks good to me!

**Pull request merged**